speech_recognition/whisper/run.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import torch
+
+
+def run_pytorch_fp32(model_name, num_runs, timeout):
+    batch_size = 1
+    sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "whisper"))
+    from utils.benchmark import run_model
+    from utils.misc import print_warning_message
+    from utils.pytorch import PyTorchRunnerV2
+    from utils.speech_recognition.libri_speech_v2 import LibriSpeech
+    from speech_recognition.whisper.whisper.whisper import load_model
+    from speech_recognition.whisper.whisper.whisper.transcribe import transcribe
+    model = load_model(model_name)
+    model.eval()
+
+    def single_pass_pytorch(_runner, _librispeech):
+        array = _librispeech.get_input_array()
+        variable_input_lengths.append(array.shape[0])
+        audio = torch.from_numpy(array.astype("float32"))
+        _librispeech.submit_transcription(_runner.run(audio)["text"].lstrip().replace(".", "").upper())
+
+    def transcribe_wrapper(audio):
+        return transcribe(model, audio, verbose=None)
+
+    runner = PyTorchRunnerV2(transcribe_wrapper)
+    librispeech = LibriSpeech()
+    variable_input_lengths = []
+    print_warning_message("Sampling rate Whisper operates at is 16,000 Hz, therefore throughput values below can be "
```

Comment:
There was a bunch of warning messages printed between this one and the results. Can we somehow print it together with the results, so it's harder to miss?